packages/client-web-kit/test/unit/core/store/id-token.spec.ts

```
@@ -19,6 +19,8 @@ function buildStore() {
                 refresh_token: 'rt',
                 id_token: 'the-id-token',
             }),
+            'POST /token/introspect': () => ({ active: true }),
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)